dateutil/utils.py

```diff
@@ -7,10 +7,11 @@
 """
 from __future__ import unicode_literals
 
-from datetime import datetime, time
+from datetime import datetime, time, tzinfo, timedelta
+from typing import Union
```

Comment:
All typing-related imports needs to go under a guard:

```python
if False:
    from typing import Union
    from datetime import tzinfo, timedelta
```

See [this](https://github.com/dateutil/dateutil/issues/383#issuecomment-396038661) for more details.

Reply:
That said I believe you don't actually need the `Union` import.